docs/cli_parameters/config/disk_config.rst

```diff
@@ -0,0 +1,244 @@
+.. _disk config:
+
+Disk Configuration
+==================
+
+There are only three modes in the ``disk_config`` option. They are described in more detail below.
```

Comment:
@Torxed shouldnt it be 4 modes? "leave as is" + the three from the guided menu (Best Effort, 
Manual Partitioning and *pre-mounted configuration*).

Or leave as is and pre-mounted configuration are the same in some way?

Reply:
> Or leave as is and pre-mounted configuration are the same in some way?

Correct :) They're identical, It's just me using different names for the same thing. I should unify those names! Thanks for pointing it out!